plugins/src/opengrok/auth/plugin/ldap/LdapFacade.java

```diff
@@ -317,11 +349,17 @@ private SearchControls prepareSearchControls() {
             for (namingEnum = server.search(dn, filter, controls); namingEnum.hasMore();) {
                 SearchResult sr = namingEnum.next();
                 reported = false;
+                if (errorTimestamp > 0) {
+                    errorTimestamp = 0;
+                    WebHook hook;
+                    if ((hook = webHooks.getRecover()) != null) {
+                        RestfulClient.postIt(hook.getURI(), hook.getContent());
```

Comment:
consider using `CompletableFuture.supplyAsync( ... );` so you don't slow down the request?